### PR TITLE
chore: fix code coverage and sonar

### PR DESCRIPTION
**Pull request by cofin**
## Description

-

### CODE REVIEW

File: CODEOWNERS
Comment (JacobCoffee) [inline CODEOWNERS:1]: I think we should keep this its what notifies us a PR is ready

Comment (cofin) [reply CODEOWNERS:1]: There is a second one in the `.github` folder.

Comment (cofin) [reply CODEOWNERS:1]: I removed this one so that it mirrors what Litestar has.

Comment (JacobCoffee) [reply CODEOWNERS:1]: oh im stupid, when you said `.github` i thought you meant https://github.com/litestar-org/.github

**Review by JacobCoffee — APPROVED** [review]

**Pull request merged**